### PR TITLE
refs #10286 - Support templates proxy feature for default PXE menu

### PR DESCRIPTION
**Pull request by stbenjam**
~~Required by https://github.com/theforeman/foreman/pull/2268~~

### CODE REVIEW

File: pxe/PXELinux_default.erb [hunk 1]
```diff
@@ -28,9 +28,9 @@ LABEL <%= "#{profile[:template]} - #{profile[:hostgroup]}" %>
      KERNEL <%= profile[:hostgroup].operatingsystem.kernel(profile[:hostgroup].architecture) %>
 <% case profile[:hostgroup].operatingsystem.pxe_type -%>
 <% when 'kickstart' -%>
-     APPEND initrd=<%= profile[:hostgroup].operatingsystem.initrd(profile[:hostgroup].architecture) %> ks=<%= default_template_url(profile[:template], profile[:hostgroup]) %> ksdevice=bootif network kssendmac
+     APPEND initrd=<%= profile[:hostgroup].operatingsystem.initrd(profile[:hostgroup].architecture) %> ks=<%= default_template_url(profile[:template], profile[:hostgroup], @proxy) %> ksdevice=bootif network kssendmac
```
Comment (domcleal) [inline pxe/PXELinux_default.erb]: This doesn't look backwards-compatible?

**Comment by stbenjam** [comment]
@domcleal How's this?

**Comment by lzap** [comment]
LGTM, will test.

**Comment by mmoll** [comment]
any test results? :)

**Comment by stbenjam** [comment]
Actually goes along with https://github.com/theforeman/foreman/pull/2327, which still needs testing

**Comment by domcleal** [comment]
Tested fine, :+1: to merge when the core PR's merged.

**Comment by ares** [comment]
@stbenjam could you please reopen against develop branch? we renamed master to develop since we started branching this repo per foreman releases

**Pull request closed without merging**